core/modules/views/tests/views_layout.test

```diff
@@ -0,0 +1,131 @@
+<?php
+/**
+ * @file
+ * Tests for Views and Layout context.
+ */
+
+require_once BACKDROP_ROOT . '/core/modules/views/tests/views_query.test';
+
+/**
+ * Tests the interface for adding, removing, and moving blocks.
+ */
+class ViewsLayoutsTest extends BackdropWebTestCase {
+
+  /**
+   * Function that handles the set-up of this test.
+  */
+  protected function setup() {
+    // Loading in required modules.
+    parent::setup(
+      'views', 'views_ui', 'layout', 'block', 'user', 'contextual'
+    );
+  }
+
+  /**
+   * Tests that a contextual filter input can be passed in from a layout.
+   */
+  public function testViewsLayouts() {
+    global $admin_user;
+
+    $permissions = array(
+      'administer site configuration',
+      'administer views',
+      'administer blocks',
+      'bypass node access',
+      'access user profiles',
+      'view revisions',
+      'synchronize configuration',
+      'administer layouts',
+      'administer flexible templates',
+    );
+
+    // Create and login as an admin with views and layouts permissions.
+    $admin_user = $this->backdropCreateUser($permissions);
+    $this->backdropLogin($admin_user);
+
+    // Navigate to the form that allows the adding of our view.
+    $this->backdropGet('admin/structure/views/add');
+
+    // Create our layout context view and configure its properties.
+    $layoutContextView = array();
+    $layoutContextView['human_name'] = 'Layout Context Test (User account)';
+    $layoutContextView['name'] = 'layout_context_test_ua';
+    $layoutContextView['show[wizard_key]'] = 'users';
+    $layoutContextView['description'] = 'This view will allow a a layout to reference contextually';
+    $layoutContextView['page[create]'] = FALSE;
+    $layoutContextView['block[create]'] = 1;
+    $layoutContextView['block[title]'] = 'block';
+    $layoutContextView['block[style][row_plugin]'] = 'fields';
+
+    $this->backdropPost('admin/structure/views/add', $layoutContextView, t('Save view'));
```

Comment:
Properties on objects are camelcase like this, but prefer to use underscores in stand-alone variable names. `$layoutContextView` => `$layout_context_view`